### DISCUSSION TITLE
Zen Web Side Panels could have a default panel

### DISCUSSION THREAD

**Discussion by juampivallejo** [zen-browser/desktop#2548] [Ideas]
I really like using Zen's Web Side Panels feature, however, it would be effortless to improve it by allowing to set a default app or website.
Currently, this is the default:
![image](https://github.com/user-attachments/assets/51f08390-00c2-465f-94a4-5dbb82eadb63)

This default does not have any use, so maybe even something simple like making the first one the default will make this feature 100 times better to me.

**Comment by tedbergstrand** [comment]
Agreed. Great idea.
